pkg/cgroups/cgroups.go

```diff
@@ -178,6 +146,10 @@ func (subsys Subsystems) AddPID(name string, pid int) error {
 	return appendIntToFile(filepath.Join(subsys.CPU, name, "cgroup.procs"), pid)
 }
 
+func (subsys Subsystems) GetPrefix() string {
```

Comment:
What is the justification for having the getter, given that both `subsys.GetPrefix()` and `subsys.Prefix` are exported? I would've expected a getter only if you had an unexported `subsys.prefix`.

nit: https://golang.org/doc/effective_go.html#Getters suggests it is not idiomatic to include `Get` in the getter name

Reply:
right. I think I wanted to keep this private but then needed to make the field public for test. I'll remove this getter.